assets/model_monitoring/components/src/model_monitor_data_joiner/run.py

```diff
@@ -51,13 +61,31 @@ def run():
     parser.add_argument("--joined_data", type=str, required=True)
     args = parser.parse_args()
 
-    join_data(
-        args.left_input_data,
+    # Load data
+    try:
+        left_input_data_df = read_mltable_in_spark(mltable_path=args.left_input_data)
+    except IndexError:
+        raise Exception('The left_input_data is empty. Please add data and try again.')
```

Comment:
Shall we include the mltable path in the message too so that users don't have to figure out which one is left_input_data?

Reply:
I think the path wouldn't be too indicative, however if we could get the MLTable name..! :O

Reply:
Yeah the path looks bad - I initially had it in the local testing. Let me know if there is a way to add the MLTable name